package.json

```diff
@@ -3,10 +3,13 @@
   "version": "4.0.3",
   "description": "`ObservableStore` is a synchronous in-memory store for a single value, that you can subscribe to updates on.",
   "main": "lib/index.js",
+  "files": [
```

Comment:
Huh, so we were publishing test files before this change?

Reply:
Yup!

```
$ tar tf obs-store-4.0.3.tgz 
package/package.json
package/.babelrc
package/.nvmrc
package/index.js
package/README.md
package/lib/asStream.js
package/lib/composed.js
package/lib/localStorage.js
package/lib/merged.js
package/lib/transform.js
package/test/basic.js
package/test/composed.js
package/test/localStorage.js
package/test/merged.js
package/test/stream.js
package/test/transform.js
```